Survey/Sources/Presentation/Modules/Login/LoginView.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  LoginView.swift
+//  Survey
+//
+//  Created by Khanh on 27/07/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Combine
+import SwiftUI
+import UIKit
+
+struct LoginView: View {
+
+    private var input: LoginViewModel.Input
+    @ObservedObject var output: LoginViewModel.Output
+
+    private let loadTrigger = PassthroughSubject<Void, Never>()
+
+    @State private var image: UIImage? = UIImage(named: "ic_background")
```

Comment:
I don't see you using this state `image`, can you explain to me?

Reply:
It's just redundant code, I removed it 🥲  at https://github.com/nkhanh44/ic-survey-ios/pull/13/commits/45ae841b35aa065645f32fb9c3e9b222f1e64540